pages/shoes/buy/[id].js

```diff
@@ -55,12 +57,14 @@ const StyledContainer = styled.section`
 
 const StyledOverviewContainer = styled.section`
   position: relative;
-  border: 2px solid black;
   width: 80vw;
-  height: auto;
+  height: 80vh;
   border-radius: 40px;
   margin-top: 10px;
-  padding: 2px;
+  padding: 5px;
+  overflow: hidden;
+  background-color: #dadada;
+  box-shadow: 0 5px 15px;
 
```

Comment:
The card seems to be cut on the bottom. You might want to try to give it some more space there.

<img width="346" alt="Screenshot 2023-01-17 at 11 29 21" src="https://user-images.githubusercontent.com/4458383/212875391-ea637714-f8f6-4cf8-ac66-0e8c96b525fa.png">

Try to add some `margin-bottom: 10px`, for example. I would prefer relative units, though (`rem` instead of `px`).

Reply:
merci :) looks much better now